### PR TITLE
Remove references to selflinks in documentation examples

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref https://github.com/kubernetes/enhancements/issues/1164

For technical approval:
/assign @liggitt

### CODE REVIEW

File: content/en/blog/_posts/2018-01-00-Introducing-Client-Go-Version-6.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ spec:
 ```
 $ kubectl create -f app.yaml
 
-The App "example-app" is invalid: []: Invalid value: map[string]interface {}{"apiVersion":"mygroup.example.com/v1", "kind":"App", "metadata":map[string]interface {}{"creationTimestamp":"2017-08-31T20:52:54Z", "uid":"5c674651-8e8e-11e7-86ad-f0761cb232d1", "selfLink":"", "clusterName":"", "name":"example-app", "namespace":"default", "deletionTimestamp":interface {}(nil), "deletionGracePeriodSeconds":(\*int64)(nil)}, "spec":map[string]interface {}{"replicas":15, "version":"v1.0.2"}}:  
+The App "example-app" is invalid: []: Invalid value: map[string]interface {}{"apiVersion":"mygroup.example.com/v1", "kind":"App", "metadata":map[string]interface {}{"creationTimestamp":"2017-08-31T20:52:54Z", "uid":"5c674651-8e8e-11e7-86ad-f0761cb232d1", "clusterName":"", "name":"example-app", "namespace":"default", "deletionTimestamp":interface {}(nil), "deletionGracePeriodSeconds":(\*int64)(nil)}, "spec":map[string]interface {}{"replicas":15, "version":"v1.0.2"}}:
```
Comment (sftim) [inline content/en/blog/_posts/2018-01-00-Introducing-Client-Go-Version-6.md]: It sounds OK to me to leave these references in blog posts. That information was appropriate at the time, right?

File: content/en/blog/_posts/2018-03-00-How-To-Integrate-Rollingupdate-Strategy.md [hunk 2]
```diff
@@ -168,8 +168,6 @@ items:
 
    resourceVersion: "415852"
 
-   selfLink: /apis/mysql.orain.com/v1/namespaces/default/mysqlclusters/clustershard-c
```
Comment (sftim) [inline content/en/blog/_posts/2018-03-00-How-To-Integrate-Rollingupdate-Strategy.md]: It sounds OK to me to leave these references in blog posts. That information was appropriate at the time, right?

File: content/en/blog/_posts/2018-01-00-Introducing-Client-Go-Version-6.md (hunk 1 above)
Comment (liggitt) [reply content/en/blog/_posts/2018-01-00-Introducing-Client-Go-Version-6.md]: it's easier to scrub all non-consequential instances

Comment (wojtek-t) [reply content/en/blog/_posts/2018-01-00-Introducing-Client-Go-Version-6.md]: +1 to Jordan - I think it's better to be consistent

**Pull request merged**